index.js

```diff
@@ -202,6 +202,45 @@ function highLight(item) {
 }
 
 /**
+ * Changes the theme based upon the value passed
+ * @param {int} themeNum - a number passed based on the button clicked by the user
+ */
+function changeTheme(themeNum) {
+    var personaSlug;
+    var personaIDs = [111387, 539838, 157076];
+
+    // set the theme slug based upon the number theme passed
+    if(themeNum === 1)
+        personaSlug = 'fox-in-snow';
+    else if(themeNum === 2)
+        personaSlug = 'cozy-fox';
+    else if(themeNum === 3)
+        personaSlug = 'shiretoko-fox';
+    // if there is no number passed, set the theme to default and return
+    else {
+        LightweightThemeManager.themeChanged(null);
+        return;
+    }
```

Comment:
Nope, I didn't catch that. I'll still need to check if themeNum is null so that I can reset the theme.